### PR TITLE
Update emulated_hue.markdown

### PR DESCRIPTION
**Pull request by martin3000**
Add an example for testing the emulated_hue integration with a curl command.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
	- Added usage examples for the `emulated_hue` integration, including practical `curl` command examples for switching lights on or off.
	- Enhanced clarity and usability of the documentation for users integrating lights with Alexa.

### CODE REVIEW

File: source/_integrations/emulated_hue.markdown [hunk 1]
```diff
@@ -161,6 +161,9 @@ You can verify that the `emulated_hue` integration has been loaded and is respon
 
 - `http://<HA IP Address>:80/description.xml` - This URL should return a descriptor file in the form of an XML file.
 - `http://<HA IP Address>:80/api/v2/lights` - This will return a list of devices, lights, scenes, groups, etc.. that `emulated_hue` is exposing to Alexa.
+  
+You can use the "curl" command to switch a light on or off:
+- `curl -X PUT -d '{"on":true}' http://<HA IP Address>/api/v2/lights/219/state` - This command switches light 219 on.
```
Comment (frenck) [inline source/_integrations/emulated_hue.markdown:166]: I'm a bit confused as to why we should add this to this documentation? What does it aid with? I can only think of it causing confusion of thinking this is the way you need to control lights?

Comment (frenck) [reply source/_integrations/emulated_hue.markdown:166]: Unresolving this comment, as there was no reply.

**Comment by martin3000** [comment]
The chapter is about "Troubleshooting" which deals with methods how to test the thing.

For troubleshooting, if something does not work, the test with "curl" can be useful.

**Comment by martin3000** [comment]
The chapter is about "Troubleshooting" which deals with methods how to test the thing.

For troubleshooting, if something does not work, the test with "curl" can be useful.

**Review by frenck — APPROVED** [review]
Aah check.

Thanks! 👍 

../Frenck

**Pull request merged**